test.old/browser.js

```diff
@@ -10,7 +10,6 @@ const pull = require('pull-stream')
 const pullGoodbye = require('pull-goodbye')
```

Comment:
Right now none of the tests here are running, just the `interface` test suite. We should just remove all of these and .aegir.js, and add tests for any utility / secio specific methods. The interface test suite is only verifying the high level api, not that the crypto exchange is correct, we should add tests for this.

Reply:
For example, it would be great to see a failing test showing the issue with temporarily using varint instead of fixed size. While it's working, we know this does not fit the spec. I'd expect to see the interface tests pass and the local tests failing there.

Reply:
Don't know how to test that without any lib supporting that. We'd need an interop test or sth.

Reply:
It will likely be easier once `it-length-prefixed` supports it, but we should be able to validate that the payload is what's being expected. Mock data could be passed to the `finish` method along with a spy on the `it-pb-rpc` wrapper to validate that when data is sent over one duplex pair half to the other, that the encrypted payload is as expected. Interop isn't really needed here, we just need to validate we're sending the right thing.